doc/source/release/0.5.0-notes.rst

```diff
@@ -30,6 +30,11 @@ Improvements
 New Features
 ------------
 
+Spatial desagragation of the input raster
+*****************************************
+
+If the resolution of the input raster is diffrent of the resolution of the model mesh, the input raster are automatically reprojected by gdal. In that case the reading of the input can be slower. For best performances, it can be usful to preprocess the input files (precipitations). Some functions available in smash.core.raster could help for pre-processing.
+
```

Comment:
Oui ca me va, je trouve ca bien et ca simplifie et clarifie l'usage. Ne te prends pas trop la tête avec la fusion avec meshing, on a le temps pour ca, c'est facile à faire, on le fera plutard. Pour le nom j'utiliserais raster_handler.py sans l'underscore stp, ca cache le fichier (c'est jamais bon de cacher des choses), c'est chiant à taper et on s'en souvient jamais.

Reply:
```suggestion
If the resolution of the input raster is different from the resolution of the model mesh, the input rasters are automatically reprojected by gdal. In that case the reading of the input can be slower. For best performances, it can be useful to preprocess the input files (precipitations). See the :ref:`API Reference <api_reference.raster_handler>` section. 
```